### ISSUE TITLE
Mark comments as blocker / non-blocker / nit / kudos

### ISSUE THREAD

**Issue by yshavit** [refined-github/refined-github#7980]
### Description

I've found that most comments fall under one of four categories:

- 🛑 blockers: really do fix this
- ❔ non-blockers: I'd like this addressed, but won't block an approval on it; but I'd appreciate at least a reply.
- 🤏 nit or stylistic: I think it'd look a bit better this way, but it's up to you
- 💜 kudos: No action needed, I just really like this bit

I'm envisioning a system where:

- Refined GitHub would provide a drop-down menu on comments, listing the above options
- If you select one, then GitHub Refined would add the appropriate emoji to your comment. For example, if I selected the kudos option and wrote "this is a great test case", the comment would be posted as `💜 this is a great test case`
- On render, GitHub Refined would strip the icon, and instead replace it with styling around the comment div itself.

(non-blockers could also be emojiless, as just the standard state of comments — but having them be with emojis is nice, because it signals to someone using this extension that the commenter explicitly marked the item as non-blocking)

This would be perfectly readable for people without GitHub Refined, and would help focus discussions on what really matters.

### Example URLs

https://github.com/yshavit/mdq/pull/191#pullrequestreview-2400006948

**Comment by fregante** [comment]
I don’t think it's worth it.

- blockers: if it's a blocker, use "Request changes", at least one comment is non-blocker
- non-blockers: just comment or pre-approve the PR, all comments are considered non-blockers
- nit: just preface it with `nit:` it's 4 characters
- kudos: just leave a heart emoji, also doable. Mine requires 2 keystrokes because it's set as a OS-wide text-replacement for "tt": 💙 

The only real advantage is being able to distinguish blocker comments from non-blocker comments when using "Request changes", but even then, you can preface it with `Blocker:`

Maintaining this UI and _convincing_ people to use these exact emojis would be too much work.

If your team already settled on this, I'd advise writing a standalone extension or userscript

**Comment by yshavit** [comment]
> The only real advantage is being able to distinguish blocker comments from non-blocker comments when using "Request changes"

If I may humbly disagree: the primary advantage would be to encourage people to think in these terms, and to remind them to label their comments. In other words, it wouldn't be so much about the UI making it easier to do something, but rather in having the UI encourage good behavior.

That said, if you still feel it's not worth it, I'll obviously respect your decision.

**Comment by fregante** [comment]
From a `netiquette` standpoint, I see the appeal, but this like `conventional-commits` can easily lead to people asking for more categories and become its own extension. 

I have to be careful what gets added here from a maintainability standpoint most of all. See the long list of features GitHub can just break with a single update: #7856

I'd be open to alternatives like `netiquette` where for example it could encourage people to either mark their review comments as nits.

As for the readability, you have to read each and every comment every way, so I don't think it's fair to the reviewer to deprioritize them visually in any way. Maybe the opposite could be useful: 

> # 🚫 Blocker
> 
> This shade of blue is not fishy enough


But then we end up in ["too harsh"](https://github.com/refined-github/refined-github/issues/1602) territory so it's just easier for the reviewer to pick whatever tone they prefer… in plain English

**Comment by yshavit** [comment]
Gotcha, that all makes sense! Thanks for replying. :)
